app/src/main/res/layout/activity_select_chatbot_activity.xml

```diff
@@ -88,219 +88,219 @@
             </androidx.constraintlayout.widget.ConstraintLayout>
         </androidx.cardview.widget.CardView>
 
-        <androidx.cardview.widget.CardView
```

Comment:
If you open satya directly from the home screen, then no changes are needed on this screen. keep this intact

Reply:
Done